tests/Fixer/Casing/MagicConstantCasingFixerTest.php

```diff
@@ -0,0 +1,119 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Tests\Fixer\Casing;
+
+use PhpCsFixer\Test\AbstractFixerTestCase;
+
+/**
+ * @author ntzm
+ *
+ * @internal
+ */
+final class MagicConstantCasingFixerTest extends AbstractFixerTestCase
+{
+    /**
+     * @param string      $expected
+     * @param null|string $input
+     *
+     * @dataProvider provideFixCases
+     */
+    public function testFix($expected, $input = null)
+    {
+        $this->doTest($expected, $input);
+    }
+
+    public function provideFixCases()
+    {
+        return array(
+            array(
+                '<?php echo __LINE__;',
+                '<?php echo __line__;',
```

Comment:
please mix up some of the cases, now you test all-lower and all-upper, having one variation would be nice (for example `__lInE__`)